frontend/src/bundles/profile/store/slice.ts

```diff
@@ -25,6 +33,30 @@ const { reducer, actions, name } = createSlice({
             state.profile = action.payload;
             state.dataStatus = DataStatus.FULFILLED;
         });
+        builder.addMatcher(isAnyOf(disconnectSocialMedia.pending), (state) => {
```

Comment:
Will there be any additional actions to handle in this case? If not, you might consider using `addCase` instead of `addMatcher`?

Reply:
Yes, it's a placeholder for the profile update action